File: 프로그래머스/1/131705. 삼총사/삼총사.js
function solution(numbers) {
    let count = 0;
    
    // 첫번째 수 
    for(let i = 0; i < numbers.length; i++){
        
        // 두번째 수 
        for(let j = i + 1; j < numbers.length; j++){
            
            // 세번째 수
            for(let k = j + 1; k < numbers.length; k++){
                
                if(numbers[i] + numbers[j] + numbers[k] === 0) count++;
            }
        }
    }
    
    console.log("삼총사", count);
    
    return count;
}

File: 프로그래머스/1/131705. 삼총사/삼총사.py
def solution(number):
    count = 0
    length = len(number)
    for x in range(length):
        for y in range(x+1, length):
            for z in range(y+1, length):
                if number[x]+number[y]+number[z] == 0:
                    count += 1
                    
    return count